delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewType.scala

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model
+
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.nxv
+
+/**
+  * Enumeration of ElasticSearch view types.
+  */
+sealed trait ElasticSearchViewType extends Product with Serializable {
```

Comment:
It seems an overkill. This is usually done just with plain Vocabulary

Reply:
I used the same structure like for resolvers... and it's useful for pattern matching.